requirements.txt

```diff
@@ -1,2 +1,2 @@
 six
-pyparsing
+pyparsing=2.1.3
```

Comment:
If you want to pin version the correct specifier is `pyparsing==2.1.3` with double  `==`, single `=` is not a valid version specifier . For more info please check [Version Specifiers on PEP 440](https://www.python.org/dev/peps/pep-0440/#version-specifiers)